jck/runtime.api/playlist.xml

```diff
@@ -16,6 +16,14 @@
 	<include>../jck.mk</include>
 	<test>
 		<testCaseName>jck_custom</testCaseName>
+		<disables>
```

Comment:
If we exclude `jck_custom` on zos, it means we cannot use it for triaging or debugging user-specified tests. Not sure if there is a better way to exclude the subtest. For example, change default `JCK_CUSTOM_TARGET` from `api/java_math` `api/java_math/BigInteger`?

Reply:
Yes. not a fan of running the whole `api/java_math` suite under `jck_custom` either - as that means we are running a target twice. 

Updated the PR to only run `api/java_math/BigInteger` - as suggested. This is faster + also works normally on z/OS , so no exclude necessary for jck_custom i: (results: Grinder_JCK/342).